crates/app/src/lib.rs

```diff
@@ -610,9 +609,7 @@ impl System<Event<'static, ()>> for ExamplesSystem {
                     },
                 ..
             } => match virtual_keycode {
```

Comment:
So these are actually used in the "native" examples, which doesn't have a Debugger. I'll merge and just revert that part

Reply:
Okay that makes a lot more sense, thank you!